main.nf

```diff
@@ -51,6 +53,7 @@ def helpMessage() {
       -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic
       --seq_center                  Text about sequencing center which will be added in the header of output bam files
       --maxMultiqcEmailFileSize     Theshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
+      --skip_mirdeep                If this parameter is specified, it stops the detection of novel miRNAs with miRDeep2. Default: false
```

Comment:
```suggestion
      --skip_mirdeep                If this parameter is specified, it stops the detection of novel miRNAs with miRDeep2. Default: ${params.skip_mirdeep}
```

You can also maybe apply this system in the rest of the help text to save you maintenance time :)